components/supervisor/frontend/src/ide/ide-metrics-service-client.ts

```diff
@@ -8,7 +8,8 @@ import { serverUrl } from '../shared/urls';
 
 export enum MetricsName {
     SupervisorFrontendClientTotal = "gitpod_supervisor_frontend_client_total",
-    SupervisorFrontendErrorTotal = "gitpod_supervisor_frontend_error_total"
+    SupervisorFrontendErrorTotal = "gitpod_supervisor_frontend_error_total",
+    SupervisorFrontendLoadTotal = "gitpod_vscode_web_load_total",
```

Comment:
We introduced a new metric: `gitpod_vscode_web_load_total` for measuring the percentage of failed loads.

Reply:
https://github.com/gitpod-io/gitpod/blob/bc4700264d318d43e71b607401eab892de2a07e2/install/installer/pkg/components/ide-metrics/configmap.go#L50-L58

this metric can have two different types: `loading` and `error`.

Reply:
supervisor-frontend should be reverted, we should try to decouple metrics in VS Code Web workbench and here, and work on them separately. I think this PR should not have any changes in supervisor. cc @mustard-mh